src/logger/loggerSampleData.c

```diff
@@ -220,6 +221,14 @@ float get_imu_sample(int channelId)
         float value = imu_read_value(channelId, c);
         return value;
 }
+
+float get_imu_gsum_getter(void)
+{
+        LoggerConfig *config = getWorkingLoggerConfig();
+        float x_value = imu_read_value(IMU_CHANNEL_X, &config->ImuConfigs[IMU_CHANNEL_X]);
+        float y_value = imu_read_value(IMU_CHANNEL_Y, &config->ImuConfigs[IMU_CHANNEL_Y]);
```

Comment:
So what about Z axis?  We do live in a 3D world after-all.

Reply:
I considered this - but it's not an industry standard thing to mix in the Z axis. I think bumps and lifts would mix in undesired noise into Gsum.